src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/core/RcaConfTest.java

```diff
@@ -63,20 +64,14 @@ public void testReadRcaConfig() {
 
   @Test
   public void testReadDeciderConfig() {
-    final Double fieldDataCacheUpperBound =
-        rcaConf.readDeciderConfig(
-            CacheDeciderConfig.CONFIG_NAME,
-            CacheDeciderConfig.RCA_CONF_KEY_CONSTANTS.FIELD_DATA_CACHE_UPPER_BOUND,
-            Double.class);
-    Assert.assertNotNull(fieldDataCacheUpperBound);
-    Assert.assertEquals(CacheDeciderConfig.DEFAULT_FIELD_DATA_CACHE_UPPER_BOUND, fieldDataCacheUpperBound, 0.01);
-
-    final Double shardRequestCacheUpperBound =
-        rcaConf.readDeciderConfig(
-            CacheDeciderConfig.CONFIG_NAME,
-            CacheDeciderConfig.RCA_CONF_KEY_CONSTANTS.SHARD_REQUEST_CACHE_UPPER_BOUND,
-            Double.class);
-    Assert.assertNotNull(shardRequestCacheUpperBound);
-    Assert.assertEquals(CacheDeciderConfig.DEFAULT_SHARD_REQUEST_CACHE_UPPER_BOUND, shardRequestCacheUpperBound, 0.01);
+    DeciderConfig configObj = new DeciderConfig(rcaConf);
+    Assert.assertNotNull(configObj.getCachePriorityOrder());
+    Assert.assertNotNull(configObj.getWorkloadPriorityOrder());
+    Assert.assertNotNull(configObj.getFieldDataCacheUpperBound());
+    Assert.assertNotNull(configObj.getShardRequestCacheUpperBound());
+    Assert.assertEquals(Arrays.asList("test-read-heavy", "test-write-heavy"), configObj.getCachePriorityOrder());
+    Assert.assertEquals(10.4, configObj.getFieldDataCacheUpperBound(), 0.01);
+    Assert.assertEquals(10.05, configObj.getShardRequestCacheUpperBound(), 0.01);
```

Comment:
0.4 and 0.05? Any reason why we are using 10.4 and 10.05?

Reply:
I have updated the values in the tmp config files so as to assert the reading from the config is working as expected.